### PR TITLE
Update styleguide.md

### PR DESCRIPTION
**Pull request by Ritu-bhangale**
From the GitHub style guide: software --> the software; copy --> a copy
Ampersands and Dates: for example:-->for example,

### Thank you for contributing! Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md
- [ ] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [x] Suggesting edits to an existing topic or collection
  - [ ] Curating a new topic or collection
  - [ ] Something that does not neatly fit into the binary options above

---------------------------------------------------------------------

### Editing an existing topic or collection

I'm suggesting these edits to an existing topic or collection:
- [ ] Image (and my file is `*.png`, square, dimensions 288x288)
- [x] Content (and my changes are in `index.md`)

> This will make a document look more proffesional.

### Curating a new topic or collection

- [ ] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [ ] My folder contains a `*.png` image (if applicable) and `index.md`
- [ ] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

> Please replace this line with an explanation of why you think this topic or collection should be curated.

### Something that does not neatly fit into the binary options above

- [ ] My suggested edits are not about an existing topic or collection, or at least not a single one
- [ ] My suggested edits are not about curating a new topic or collection, or at least not a single one
- [ ] My suggested edits conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

> Please replace this line with an explanation of your proposed changes.

Minor grammatical errors.

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

File: docs/styleguide.md [hunk 1]
```diff
@@ -2,9 +2,9 @@
 
 From the GitHub Style Guide:
 
-> Words are an important part of how software works. Just as we have a style guide for our code, we have a style guide for our tone and our voice. Even though there may be dozens of people creating a product, it should still sound like we speak in one consistent voice.
+> Words are an important part of how the software works. Just as we have a style guide for our code, we have a style guide for our tone and our voice. Even though there may be dozens of people creating a product, it should still sound like we speak in one consistent voice.
```
Comment (mvkaran) [inline docs/styleguide.md:5]: Thanks for your contribution! In this scenario, the word "software" implies not just the actual code comprising of software, but also the development, usage, maintenance etc. Also, we are not talking of a single piece of software to necessitate using "the". The current form is correct :)

Comment (mvkaran) [inline docs/styleguide.md:5]: Thanks for your contribution! In this scenario, the word "software" implies not just the actual code comprising of software, but also the development, usage, maintenance etc. Also, we are not talking of a single piece of software to necessitate using "the". The existing form of "how software works" is correct :)

File: docs/styleguide.md [hunk 2]
```diff
@@ -2,9 +2,9 @@
...
 >
->**The way we write is just as important as the way we build. Consider these things when writing copy.**
+>**The way we write is just as important as the way we build. Consider these things when writing a copy.**
```
Comment (mvkaran) [inline docs/styleguide.md:7]: "Copy" here doesn't imply a "duplicate", but a well-known short form for "copywrite" or "copywriting". Hence, the current form of "writing copy" or "copy writing" is correct :)

**Review by mvkaran — CHANGES_REQUESTED** [review]

**Comment by kenyonj** [comment]
Closing this PR as @mvkaran addressed the changes and how they conflict with our intentions in the current copy.

**Pull request closed without merging**